YANG/ccamp/otn-topology/otn-topology-example.json.folded

```diff
@@ -0,0 +1,307 @@
+=============== NOTE: '\' line wrapping per RFC 8792 ================
+
+{
+  "ietf-network:networks": {
+    "network": [
+      {
+        "network-id": "example:otn-topology-example",
+        "network-types": {
+          "ietf-te-topology:te-topology": {
+            "ietf-otn-topology": {}
+          }
+        },
+        "ietf-te-topology:te-topology-identifier": {
+          "topology-id": "otn-topology-example"
+        },
+        "ietf-te-topology:te": {},
+        "node": [
+          {
+            "node-id": "example:D1",
+            "ietf-te-topology:te-node-id": "192.0.0.1",
+            "ietf-te-topology:te": {
+              "te-node-attributes": {
+                "name": "Node D1"
+              }
+            },
+            "ietf-otn-topology:otn-node": {},
```

Comment:
As in the previous example  otn-node should be under te-node-attributes

Reply:
You are right

I have validated the JSON example with yanglint in https://github.com/haomianzheng/IETF-ACTN-YANG-Model/pull/184/commits/62acf616649e7b751894db8aa8f6a0d977acc7ce